index.js

```diff
@@ -2,5 +2,5 @@
 
 import createRunner from "./lib/create-runner";
 
-module.exports = createRunner();
-module.exports.createRunner = createRunner;
+exports.default = createRunner();
```

Comment:
```suggestion
exports.default = createRunner;
```

Reply:
form the [docs](https://flight-manual.atom.io/hacking-atom/sections/writing-specs/#customizing-your-test-runner):

> Your test runner module must export a single function, which Atom will call within a new window to run your package's tests.

the default export is so people can use `"atomTestRunner": "atom-jasmine3-test-runner"` if they want to use the default configuration.